wp-content/themes/borderzine/inc/widgets/class-borderzine-6-col-widget.php

```diff
@@ -0,0 +1,301 @@
+		$instance = $old_instance;
+		$instance['title'] = sanitize_text_field( $new_instance['title'] );
+		$instance['num_posts'] = intval( $new_instance['num_posts'] );
+		$instance['avoid_duplicates'] = ! empty( $new_instance['avoid_duplicates'] ) ? 1 : 0;
+		$instance['thumbnail_display'] = sanitize_key( $new_instance['thumbnail_display'] );
+		$instance['image_align'] = sanitize_key( $new_instance['image_align'] );
+		$instance['excerpt_display'] = sanitize_key( $new_instance['excerpt_display'] );
+		$instance['num_sentences'] = intval( $new_instance['num_sentences'] );
+		$instance['show_byline'] = ! empty($new_instance['show_byline']);
+		$instance['hide_byline_date'] = ! empty($new_instance['hide_byline_date']);
+		$instance['show_top_term'] = ! empty($new_instance['show_top_term']);
+		$instance['show_icon'] = ! empty($new_instance['show_icon']);
+		$instance['cat'] = intval( $new_instance['cat'] );
+		$instance['tag'] = sanitize_text_field( $new_instance['tag'] );
+		$instance['taxonomy'] = sanitize_text_field( $new_instance['taxonomy'] );
+		$instance['term'] = sanitize_text_field( $new_instance['term'] );
+		$instance['author'] = intval( $new_instance['author'] );
+		$instance['linktext'] = sanitize_text_field( $new_instance['linktext'] );
+		$instance['linkurl'] = esc_url_raw( $new_instance['linkurl'] );
+		return $instance;
+	}
+
+	function form( $instance ) {
+		$defaults = array(
+			'title' => __( 'Recent ' . of_get_option( 'posts_term_plural', 'Posts' ), 'largo' ),
+			'num_posts' => 5,
+			'avoid_duplicates' => '',
+			'thumbnail_display' => 'small',
+			'image_align' => 'left',
+			'excerpt_display' => 'num_sentences',
+			'num_sentences' => 2,
+			'show_byline' => '',
+			'hide_byline_date' => '',
+			'show_top_term' => '',
+			'show_icon' => '',
+			'cat' => 0,
+			'tag' => '',
+			'taxonomy' => '',
+			'term' => '',
+			'author' => '',
+			'linktext' => '',
+			'linkurl' => ''
+		);
+		$instance = wp_parse_args( (array) $instance, $defaults );
+		$duplicates = $instance['avoid_duplicates'] ? 'checked="checked"' : '';
+		$showbyline = $instance['show_byline'] ? 'checked="checked"' : '';
+		$hidebylinedate = $instance['hide_byline_date'] ? 'checked="checked"' : '';
+		$show_top_term = $instance['show_top_term'] ? 'checked="checked"' : '';
+		$show_icon = $instance['show_icon'] ? 'checked="checked"' : '';
+		?>
+
+		<p>
+			<label for="<?php echo $this->get_field_id( 'title' ); ?>"><?php _e( 'Title:', 'largo' ); ?></label>
+			<input id="<?php echo $this->get_field_id( 'title' ); ?>" name="<?php echo $this->get_field_name( 'title' ); ?>" value="<?php echo $instance['title']; ?>" style="width:90%;" />
+		</p>
+
+		<p>
+			<label for="<?php echo $this->get_field_id( 'num_posts' ); ?>"><?php _e('Number of posts to show:', 'largo'); ?></label>
+			<input id="<?php echo $this->get_field_id( 'num_posts' ); ?>" name="<?php echo $this->get_field_name( 'num_posts' ); ?>" value="<?php echo $instance['num_posts']; ?>" style="width:90%;" />
```

Comment:
should be `input type="number"`, with `step="6"` and `min="6"` to match the gallery's expected size. See https://github.com/INN/umbrella-borderzine/pull/67/files#diff-67cd39194ce850d719685ba86cf17fbfR235

Not your fault; the Largo Recent Posts widget is old and in need of cleanup, and https://github.com/INN/umbrella-borderzine/pull/67/ is a step towards that cleanup for https://github.com/INN/largo/issues/1807. I should've mentioned it earlier.

Reply:
@benlk Updated in https://github.com/INN/umbrella-borderzine/pull/75/commits/f766833d9593870549f624102f4c410abd663258. Thanks for the heads up.